docs/manual/shared_memory/shared_mem_config.rst

```diff
@@ -0,0 +1,110 @@
+.. include:: ../external-links.part.rst
+
+.. index::
+    single: Shared memory; Configuration
+
+.. _shared_mem_config:
+
+Shared memory configuration
+===========================
+
+There are two levels of configuration for |var-project-short| with shared memory 
+support, the shared memory service (|url::iceoryx_link|) level, and the 
+|var-project-short| level.
+
+The performance of the shared memory service is strongly dependent on how well 
+its configuration matches up with the use-cases it is asked to support. By configuring 
+shared memory correctly, large gains in performance can be made.
+
+The memory of iceoryx is layed out as numbers of fixed-size segments, which are 
+iceoryx's memory pool. When a subscriber requests a block of memory from iceoryx, the 
+smallest block that fits the requested size (and is available) is provided to the 
+subscriber from the pool. If no blocks can be found that satisfy these requirements, 
+the publisher requesting the block gives an error and aborts the process.
+
+.. note::
+  For testing, the default memory configuration usually suffices. 
+
+The default configuration has blocks in varying numbers and sizes up to 4MiB. iceoryx 
```

Comment:
```suggestion
The default configuration has blocks in varying numbers and sizes up to 4MiB. Iceoryx 
```

Reply:
I think that iceoryx should remain lowercase. On its own website, it shows it as iceoryx.

https://iceoryx.io/v2.0.2/getting-started/what-is-iceoryx/

Reply:
It definitely removes confusion with an 'l' with those silly fonts that have very similar 'I' and 'l'.